### PR TITLE
Check DOWNLOAD_PATH (when set) in OK Computer

### LINKED ISSUE #320: Settings.DOWNLOAD_PATH is configured but not used by is_it_working
In production, `Settings.DOWNLOAD_PATH` is configured but not used by is_it_working.

See https://github.com/sul-dlss/earthworks/blob/master/config/initializers/is_it_working.rb#L6

### PR DESCRIPTION
**Pull request by jkeck**
Closes #320

### CODE REVIEW

File: config/initializers/okcomputer.rb [hunk 1]
```diff
@@ -2,4 +2,10 @@
 OkComputer.mount_at = false
 
 OkComputer::Registry.register "solr", OkComputer::HttpCheck.new(Blacklight.default_index.connection.uri.to_s.sub(/\/$/, '') + "/admin/ping")
-OkComputer::Registry.register "downloads-cache", OkComputer::DirectoryCheck.new((Rails.root + "tmp/cache/downloads").to_s, true)
+
+if Settings.DOWNLOAD_PATH
+  OkComputer::Registry.register "downloads-cache", OkComputer::DirectoryCheck.new(Settings.DOWNLOAD_PATH, true)
```
Comment (jkeck) [inline config/initializers/okcomputer.rb:7]: I'm a _weee_ bit nervous about this since we don't really have a way to test this in stage AFAICT.

Comment (mejackreed) [reply config/initializers/okcomputer.rb:7]: Could we edit the config on the server to test?

Comment (jkeck) [reply config/initializers/okcomputer.rb:7]: Sure, would you suggest temporarily setting a directory on stage that is diff from the temp dir?

Prod's dir is in `/var/cache`, should we just use the same dir on stage (permanently or temporarily)?

Comment (mejackreed) [reply config/initializers/okcomputer.rb:7]: I'm not sure we want to set it to the same as prod.. just maybe even `/tmp/foo` and create that ?

Comment (jkeck) [reply config/initializers/okcomputer.rb:7]: Okay, that seems to have worked on stage when I deployed this branch, configed a DOWNLOAD_PATH in tmp, and added the directory.

File: config/initializers/okcomputer.rb [hunk 2]
```diff
@@ -2,4 +2,10 @@
...
+else
+  # What geoblacklight uses when no DOWNLOAD_PATH is set
+  OkComputer::Registry.register "downloads-cache", OkComputer::DirectoryCheck.new((Rails.root + "tmp/cache/downloads").to_s, true)
+end
```
Comment (cbeer) [inline config/initializers/okcomputer.rb:11]: 🤷‍♂️ 

```suggestion
  OkComputer::Registry.register "downloads-cache", OkComputer::DirectoryCheck.new(Settings.DOWNLOAD_PATH || (Rails.root + "tmp/cache/downloads").to_s, true)
```

**Pull request merged**